tests/python/contrib/test_onnx.py

```diff
@@ -0,0 +1,467 @@
+            func = relay.Function([x], y)
+            x_data = np.random.uniform(size=shape).astype(dtype)
+            verify_results(func, [x_data], 'test_softmax', rtol=1e-5, atol=1e-5)
+
+    verify_sofmax()
+
+
+def test_squeeze():
+    def verify_squeeze(shape, dtype, axis):
+        x = relay.var("x", relay.TensorType(shape, dtype))
+        z = relay.squeeze(x, axis=axis)
+        func = relay.Function([x], z)
+        x_data = np.random.random_sample(shape).astype(dtype)
+        verify_results(func, [x_data], 'test_squeeze', rtol=1e-5, atol=1e-5)
+
+    verify_squeeze((1, 3, 2, 5), "float32", None)
+    verify_squeeze((1, 3, 1), "float32", [2, ])
+    verify_squeeze((1, 2, 1, 2, 1), "float32", [0, 2])
+
+
+def test_mean():
+    def verify_mean(data_shape, axis, exclude, keepdims):
+        dtype = "float32"
+        x = relay.var('x', shape=data_shape, dtype=dtype)
+        y = relay.mean(x, axis, keepdims, exclude)
+        func = relay.Function([x], y)
+        x_data = np.random.uniform(size=data_shape).astype(dtype)
+        verify_results(func, [x_data], 'test_mean', rtol=1e-5, atol=1e-5)
+
+    verify_mean((1, 2), 0, False, False)
+    verify_mean((1, 2), 0, True, False)
+    verify_mean((1, 2), 0, True, True)
+    verify_mean((1, 2), 1, True, True)
+    verify_mean((3, 2, 1), 1, False, True)
+
+
+def test_split():
+    def verify_split(dshape, indices_or_sections, axis=None):
+        dtype = "float32"
+        x = relay.var("x", relay.ty.TensorType(dshape, "float32"))
+        y = relay.split(x, indices_or_sections, axis=axis)
+        func = relay.Function([x], y.astuple())
+        x_data = np.random.uniform(size=dshape).astype(dtype)
+
+        verify_results(func, [x_data], 'test_split', rtol=1e-5, atol=1e-5)
+
+    verify_split((5, 5, 2, 2), 5, axis=1)
+    verify_split((5, 5, 2, 2), 5, axis=0)
+    verify_split((5, 5, 2, 2), [1, 3, 4], axis=0)
+    verify_split((5, 5, 2, 2), [1, 3, 4], axis=1)
+
+
+def test_concatenate():
+    def verify_concatenate(shapes, axis, dtype="float32"):
+        in_vars = []
+        in_data = []
+        for i, shape in enumerate(shapes):
+            in_vars.append(relay.var("x" + str(i), relay.ty.TensorType(shape, dtype)))
+            in_data.append(np.random.uniform(size=shape).astype(dtype))
+
+        out_tensor = relay.concatenate(in_vars, axis)
+        func = relay.Function(in_vars, out_tensor)
+        verify_results(func, in_data, 'test_concatenate', rtol=1e-5, atol=1e-5)
+
+    verify_concatenate([(2,), (2,), (2,)], -1)
+    verify_concatenate([(2, 3, 4), (2, 2, 4), (2, 5, 4)], 1)
+    verify_concatenate([(1, 2, 4), (1, 2, 3), (1, 2, 7), (1, 2, 8), (1, 2, 1)], -1)
+    verify_concatenate([(5, 6, 7, 3),
+                        (16, 6, 7, 3),
+                        (12, 6, 7, 3),
+                        (8, 6, 7, 3),
+                        (2, 6, 7, 3)], 0)
+    verify_concatenate([(1, 14400), (1, 2400), (1, 640), (1, 240)], 1)
+
+
+def test_strided_slice():
```

Comment:
This test fails due to the change in #4312.  Could you rebase onto the latest master?

Reply:
Changed implementation as per new changes.